docs/modules/ROOT/pages/tests.adoc

```diff
@@ -14,7 +14,7 @@ For the test source code, see https://github.com/datastax/ragstack-ai/tree/main/
 
 RAGStack tests are published to https://ragstack-ai.testspace.com/[Testspace]{external-link-icon}.
 
-Tests are run multiple times daily against DSE (Cassandra) and AstraDB (Cassandra as a Service) vector databases.
+Tests are run multiple times daily against DSE (Cassandra) and Astra DB (Cassandra as a Service) vector databases.
```

Comment:
This sentence is confusing and doesn't match the company messaging about Astra DB Serverless. Why is Cassandra in parentheses after DSE? Also, why is Cassandra as a Service in parentheses after Astra DB? For Astra DB Serverless, an effort was made to disassociate Cassandra from Astra DB.

Reply:
We can use just use AstraDB and DSE

Reply:
Fixed in next commit.